### DISCUSSION TITLE
no changes are displayed

### DISCUSSION THREAD

**Discussion by ukraine-sdg** [open-sdg/open-sdg#2083] [Q&A] [answered]
Hello! 
The data from the "meta" folder on the test environment is not being updated. For example, indicator 4.4.3 (https://ukraine-sdg.github.io/site/4-4-3/)

**Comment by sarah-beasley** [comment, marked as answer]
Hi, it looks like your site isn't rebuilding which is probably due to the gem install bundler error that is occurring across all platforms. Can you try fixing this in the site repository by following these steps - https://github.com/open-sdg/open-sdg/discussions/2080 and hopefully then the changes you make in your data repository will appear on your site. Let me know if you have any problems!

**Reply by ukraine-sdg** [reply]
Thank you very much for your reply. After the changes were made, the data was updated.
